ncm-network/src/main/perl/network.pm

```diff
@@ -1145,6 +1312,9 @@ sub Configure
         my @op;
 
         while (my ($k, $v) = each(%$opts)) {
+            if ($v->isType(LIST)) {
```

Comment:
i'm surprised this works. i'm going to assume you use a recent CCM?

Reply:
It may not work :-) The original code which we still use is much uglier, and I just found this construct in one of the other components and decided it's not _that_ horrible. I think you already have a much more generic solution in #1087, but I did not have time to try that so far.

Reply:
the `->getTree` call a few lines higher returns arrayrefs for lists, that's why i was asking.